js/addresses.js

```diff
@@ -0,0 +1,45 @@
+let TapyrusApi = require('tapyrus_api');
+
+// Set your Access Token
+let access_token = ""
+
+let defaultClient = TapyrusApi.ApiClient.instance;
+defaultClient.defaultHeaders = { Authorization: `Bearer ${access_token}` }
```

Comment:
I think it is no problem here so far. It is good to consider improving the authentication method in later steps.